src/_layouts/rss2-feed.njk

```diff
@@ -48,10 +48,10 @@
 		<pubDate>{{ post.date.toUTCString() }}</pubDate>
 		<dc:creator>{{ metadata.author.name }}</dc:creator>
 		<guid isPermaLink="true">{{ absolutePostUrl }}</guid>
-		<description>{{ post.data.excerpt }}</description>
+		<description>{{ post.data.excerpt or "???" }}</description>
 		<content:encoded><![CDATA[{{ post.templateContent | htmlToAbsoluteUrls(absolutePostUrl) | safe }}]]></content:encoded>
 		{%- for tag in post.tags %}
-		<category><![CDATA[tag]]></category>
+		<category><![CDATA[{{tag}}]]></category>
```

Comment:
Can't remember why I changed this. I don't think it broke anything. Just looked funny without the `{{...}}`, like it might output the word "tag". I checked the /feed.xml file and didn't see any `<category>` elements anyways. 🤷